### DISCUSSION TITLE
HACS update

### DISCUSSION THREAD

**Discussion by sky-jreves** [leikoilja/ha-google-home#319] [Q&A]
Am I the only one getting issues after HACS 15.0.0 update?
After updating it yesterday my sensors stopped updating, I also removed and added the integration again and the sensors weren't even created, only the device were.
Anyone else with this issue?

**Comment by leikoilja** [comment]
Hey, @sky-jreves, this doesn't seem to be a general problem. If the integration does not work for you, please open a new github issue following all the recommendations in the issue template (collect and provide all the logs) :)
